Cargo.lock

```diff
@@ -1765,7 +1765,7 @@ checksum = "79386fdcec5e0fde91b1a6a5bcd89677d1f9304f7f986b154a1b9109038854d9"
 dependencies = [
  "az",
  "bytemuck",
- "half 1.8.2",
+ "half 2.3.1",
```

Comment:
unrelated change?

Reply:
I'm not sure why it decided to do this, I only added `serde_yaml` as a dev dependency. We're already on Rust 1.72, and `fixed` has its `half` version set as `>=1.8,<3`, so it seems harmless.